vyper/parser/stmt.py

```diff
@@ -194,14 +190,19 @@ def _assert_reason(self, test_expr, msg):
         return LLLnode.from_list(assert_reason, typ=None, pos=getpos(self.stmt))
 
     def parse_Assert(self):
-        with self.context.assertion_scope():
```

Comment:
This was the assert_modifiable scope, right?

Reply:
Yep